packages/trip-details/src/index.js

```diff
@@ -137,8 +195,14 @@ function TripDetails({
 TripDetails.propTypes = {
   /** Used for additional styling with styled components for example. */
   className: PropTypes.string,
+  /** Determines which transit fare should be displayed by default, should there be multiple transit fare types */
+  configDefaultFare: PropTypes.string,
```

Comment:
Why not `defaultFareKey`?

Reply:
Renamed in 088cc5ff030b873cf8e3b93a64a5a84377ce7142